test/integration/targets/beadm/tasks/beadm.yml

```diff
@@ -0,0 +1,158 @@
+# Creation of boot environment
+- name: Create new 'test-be' ZFS boot environment
+  beadm:
+    name: test-be
+    state: present
+  register: beadm_create_result
+
+- debug: var=beadm_create_result
+
+- name: Get current boot environments
+  shell: beadm list -H
+  register: boot_envs
+
+- name: Verify if 'test-be' is created
+  assert:
+    - "beadm_create_result.rc == 0"
```

Comment:
No need to quote this line.

- We tend to only quote in YAML
- We tend to only indent the minimal needed (2 spaces, no indent for lists).

Both of these are recommendations to make examples in documentation unified (and to make it more clear what YAML requires). They do not apply to integration tests, I justed wanted to inform you. (We have lots of examples doing it differently)